pkg/architecture/arch.go

```diff
@@ -2,12 +2,16 @@ package architecture
 
 import "fmt"
 
+const ArchS390x = "s390x"
```

Comment:
Hi @0xFelix ,

Previously, I was using the `case "s390x": return "s390x"` directly inside `GetImageArchitecture`. I added it as a constant after the CI showed an error related to `golangci-lint` on Friday. The error message was:

```
pkg/architecture/arch.go:11:7: string s390x has 2 occurrences, make it a constant (goconst)
	case "s390x":
	     ^
```


Sure, I can revert to the previous format and add the comment `//nolint:goconst` to avoid the linter error about the repeated string. Please let me know if you would prefer this approach. 
Thank you!

Reply:
I have updated the file. Thanks :)